configurator/config/config.go

```diff
@@ -51,6 +51,7 @@ func ParseConfig() (c PMMConfig) {
 	flag.Parse() // command line should overide config
 	c.setDefaultValues()
 	c.setLogger()
+	c.validateValues()
```

Comment:
I would say that validation should validate values and exit with errors, not silently fix them.